src/TraceEvent/RegisteredTraceEventParser.cs

```diff
@@ -979,6 +996,21 @@ private DynamicTraceEventData.PayloadFetchClassInfo ParseFields(int startField,
             ref int infoSize
         );
 
+        [DllImport("tdh.dll")]
+        internal static extern int TdhEnumerateManifestProviderEvents(
+            [In] ref Guid Guid,
```

Comment:
We can actually use `in` instead of `ref` now with P/Invokes so that the caller doesn't have to use the ref keyword.

Reply:
See, for example, what we do with the provider Guid in `EnableTrace` in TraceEventNativeMethods.cs.